ros/carma_autoware_build

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2019 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# This scipt builds the CARMA platform and its Autoware.ai dependancies. 
+
+usage() { 
+  echo "USAGE carma_build [OPTION]
```

Comment:
Should this be updated to "carma_autoware_build"? Might be easier to reference `$0` here just to get the filename, in case we rename it again.